src/nft/hooks/useCollectionFilters.ts

```diff
@@ -0,0 +1,97 @@
+import create from 'zustand'
+import { devtools } from 'zustand/middleware'
+
+export enum SortBy {
+  LowToHigh,
+  HighToLow,
+  RareToCommon,
+  CommonToRare,
+}
+
+export const SortByPointers = {
+  [SortBy.HighToLow]: 'highest',
```

Comment:
Do these need translations?

Reply:
We have no started localization, that will be handled in a subsequent PR